perception/object_recognition/prediction/map_based_prediction/src/map_based_prediction.cpp

```diff
@@ -295,7 +295,7 @@ void MapBasedPrediction::getLinearPredictedPath(
   for (double dt = 0.0; dt < time_horizon + ep; dt += sampling_delta_time) {
     geometry_msgs::msg::PoseWithCovarianceStamped pose_cov_stamped;
     pose_cov_stamped.header = origin_header;
-    pose_cov_stamped.header.stamp = rclcpp::Time(origin_header.stamp) + rclcpp::Duration(dt);
+    pose_cov_stamped.header.stamp = rclcpp::Time(origin_header.stamp) + rclcpp::Duration::from_seconds(dt);
```

Comment:
Could you apply lint-format?

Reply:
Seems like `mpc_follower` does not include `ament_lint_common`. Will fix them